build.sh

```diff
@@ -0,0 +1,35 @@
+mkdir logs
+
+echo "Running expression benchmarks"
```

Comment:
Conventionally we tend to prefix these informational lines with `===>`  so that they stand out from other output.

Reply:
Fixed.